src/main/webapp/WEB-INF/spring/test/applicationTestContext.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="
+              http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd">
+
+    <import resource="file:src/main/webapp/WEB-INF/spring/applicationContext-jpa.xml"/>
+
+    <bean id="letterToAllophoneMappingsController"
+          class="ai.elimu.rest.v2.crowdsource.LetterToAllophoneMappingsController"/>
+</beans>
```

Comment:
Is including this file a requirement for enabling the Spring mock (`MockMvcRequestBuilders`) functionality?